crates/bitwarden-core/src/key_management/crypto.rs

```diff
@@ -166,7 +164,7 @@ pub(super) async fn initialize_user_crypto(
         client.internal.init_user_id(user_id)?;
     }
 
-    let key_state = (&req).into();
+    let key_state = req.account_cryptographic_state.clone();
```

Comment:
This clone doesn't seem needed
```suggestion
    let key_state = req.account_cryptographic_state;
```